packages/in_app_purchase/ios/Tests/Stubs.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+#import <StoreKit/StoreKit.h>
+#import "FIAPReceiptManager.h"
+#import "FIAPRequestHandler.h"
+#import "InAppPurchasePlugin.h"
+
+NS_ASSUME_NONNULL_BEGIN
+API_AVAILABLE(ios(11.2), macos(10.13.2))
+@interface SKProductSubscriptionPeriodStub : SKProductSubscriptionPeriod
+- (instancetype)initWithMap:(NSDictionary *)map;
+@end
+
+API_AVAILABLE(ios(11.2), macos(10.13.2))
+@interface SKProductDiscountStub : SKProductDiscount
+- (instancetype)initWithMap:(NSDictionary *)map;
+@end
+
+@interface SKProductStub : SKProduct
+- (instancetype)initWithMap:(NSDictionary *)map;
+@end
+
+@interface SKProductRequestStub : SKProductsRequest
+- (instancetype)initWithProductIdentifiers:(NSSet<NSString *> *)productIdentifiers;
+- (instancetype)initWithFailureError:(NSError *)error;
+@end
+
+@interface SKProductsResponseStub : SKProductsResponse
+- (instancetype)initWithMap:(NSDictionary *)map;
+@end
+
+@interface InAppPurchasePluginStub : InAppPurchasePlugin
+@end
+
+@interface SKPaymentQueueStub : SKPaymentQueue
+@property(assign, nonatomic) SKPaymentTransactionState testState;
+@end
+
+@interface SKPaymentTransactionStub : SKPaymentTransaction
+- (instancetype)initWithMap:(NSDictionary *)map;
+- (instancetype)initWithState:(SKPaymentTransactionState)state;
+@end
+
+@interface SKMutablePaymentStub : SKMutablePayment
+- (instancetype)initWithMap:(NSDictionary *)map;
+@end
+
+@interface NSErrorStub : NSError
+- (instancetype)initWithMap:(NSDictionary *)map;
+@end
+
+@interface FIAPReceiptManagerStub : FIAPReceiptManager
+@end
+
+API_AVAILABLE(ios(7.0), macos(10.9))
```

Comment:
Flutter supports 8 and up so this might not be necessary. I'm ok leave it though.

Reply:
I'll remove it, most of the others should also have `API_AVAILABLE(ios(7.0), macos(10.9))` so this isn't consistent.